geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/configuration/ConnectorService.java

```diff
@@ -106,47 +89,16 @@
  */
 @Experimental
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(name = "", propOrder = {"connection", "regionMapping"})
+@XmlType(name = "", propOrder = {"regionMapping"})
 @XmlRootElement(name = "connector-service", namespace = "http://geode.apache.org/schema/jdbc")
 @XSDRootElement(namespace = "http://geode.apache.org/schema/jdbc",
     schemaLocation = "http://geode.apache.org/schema/jdbc/jdbc-1.0.xsd")
 public class ConnectorService implements CacheElement {
-  @XmlElement(namespace = "http://geode.apache.org/schema/jdbc")
```

Comment:
Note: deleted jdbc xml namespace.

Reply:
My understanding of line 114 was that this XmlElement annotation applied to the next line which was: protected List<ConnectorService.Connection> connection;
But we deleted the "connection" line.
Originally we left line 114 in and the code failed to compile.
So I'm going to resolve this. I don't understand these xml annotations well so let me know if I have this wrong.